packages/iTwinUI-react/src/core/Toast/Toaster.tsx

```diff
@@ -30,6 +31,11 @@ export type ToasterSettings = {
     | 'bottom'
     | 'bottom-start'
     | 'bottom-end';
+  /**
+   * If using React v18, use`ReactDom.createRoot`function to return root and enable v18.
+   * Otherwise, old `ReadDom.render` method will be used to render toasts.
+   */
+  getRoot?: (container: Element | undefined) => Root;
```

Comment:
Why is this a function instead of just `Root`?

Also, maybe this could be named better. `getRoot` doesn't make it obvious root for what. And the JSDoc also doesn't explain. But I don't have a better suggestion yet.

Reply:
removed